src/commands/devchain.js

```diff
@@ -54,7 +54,8 @@ exports.task = async function ({ port = 8545, reset = false, showAccounts = 2 })
       const server = ganache.server({
         gasLimit: BLOCK_GAS_LIMIT,
         mnemonic: MNEMONIC,
-        db_path: snapshotPath
+        db_path: snapshotPath,
+        network_id: Math.floor(Math.random() * 1000)
```

Comment:
As @sohkai points out, the `networkId` that is used to generate the snapshot is `15`. We should keep using 15 to avoid potential issues.